specs/Standards/NonFungibleToken/Core.md

```diff
@@ -163,7 +162,6 @@ The following behavior is required, but contract authors may name this function
 // Returns true if token was successfully transferred to `receiver_id`.
 function nft_resolve_transfer(
   owner_id: string,
-  receiver_id: string,
```

Comment:
This is local callback, so it doesn't need to be here in the Standard.

Reply:
We use `receiver_id`, because the `nft_resolve_transfer` has the ability to revert the transfer in case the `nft_on_transfer` panics or returns `false`. `receiver_id` is currently used to make sure we rollback from the correct receiver. So if the receiver has changed again, the rollback can't be completed.

Reply:
So basically something like this

impl MintbaseStore {
  #[private]
  pub fn nft_resolve_transfer(
    &mut self,
    owner_id: String,
    token_id: u64,
    approvals: HashMap<AccountId, u64>,
    receiver_id: AccountId,
    memo: Option<String>,
  ) {
    assert_eq!(receiver_id, self.nft_owner(token_id.to_string()).unwrap());